tests/render/test_render_BulletListContentBlock.py

```diff
@@ -60,22 +62,46 @@ def test_parse_row_condition_string_pandas_engine():
 @pytest.mark.smoketest
 @pytest.mark.rendered_output
 def test_all_expectations_using_test_definitions():
-    test_files = glob.glob("tests/test_definitions/*/expect*.json")
-
-    all_true = True
-    failure_count, total_count = 0, 0
-    types = []
+    # Chetan - 20220129 - During v0.14.4, it was revealed that this test was broken.
+    # The `glob` statement did not pick up any relevant tests, resulting in `test_files` being empty.
+    # Without anything to iterate over, the test gave the impression of passing (when in actuality, it never tested anything).
+    #
+    # After some research, it seems as though this has been broken since the v0.13.0 release.
+    # The 5 Expectations noted below are implemented or updated after v0.13.0 and are incompatible with this test fixture due to
+    # having incomplete render methods.
+    #
+    # As this behavior is implemented, the `UNSUPPORTED_EXPECTATIONS` list will be updated to reflect GE's current capabilities.
+
+    dir_path = os.path.dirname(os.path.abspath(__file__))
+    pattern = os.path.join(
+        dir_path, "..", "..", "tests/test_definitions/*/expect*.json"
```

Comment:
You should probably split up the other parts of the last arg to os.path.join as well

Reply:
good call!